src/main/java/cceclipseplugin/editor/listeners/DocumentChangeListener.java

```diff
@@ -99,14 +101,14 @@ public void documentAboutToBeChanged(DocumentEvent event) {
 
 		// If no diffs left; abort
 		if (newDiffs.isEmpty()) {
-			System.out.println("No new diffs, aborting.");
+			Logger.getInstance().log(IStatus.INFO, "No new diffs, aborting");
 			return;
 		}
 
 		// Create the patch
 		Patch patch = new Patch(fileMeta.getVersion(), newDiffs);
 
-		System.out.println("DocumentManager sending change request");
+		Logger.getInstance().log(IStatus.INFO, "DocumentManager sending change request");;
```

Comment:
I've noticed that most of these messages that are not errors or warnings are set to `IStatus.INFO`. Do you have a reason for making these 'info' vs. 'debug'? My first thought is to make it the 'debug' type, since they're basically 'debug statements'.